packages/compass-indexes/src/modules/create-index/index.ts

```diff
@@ -223,8 +267,8 @@ export const createIndex = () => {
 
     if (state.useColumnstoreProjection) {
       try {
-        // columnstoreProjection is not part of CreateIndexesOptions yet
-        (options as any).columnstoreProjection = EJSON.parse(
+        // @ts-expect-error columnstoreProjection is not a part of CreateIndexesOptions yet.
```

Comment:
@lerouxb here https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/expiring-todo-comments.md#dependency-version would be awesome, we could make this expire on the next version of driver till driver has it and however did the update can fix it. Would you add a link to this comment in the ticket to add the eslint rule?